[DOC-1371]/[DOC-1372] add note on launch wait and fix typos (#1647)
spatialos/UnrealGDK: SpatialGDK/Documentation/content/get-started/example-project/exampleproject-local-deployment.md

There are two types of deployment: local and cloud.

- A **local deployment** is for testing only. A local deployment runs your game simulation in your Unreal Editor. 

> </br>Your game simulation with its own instance of the SpatialOS Runtime, runs on your development machine. You can run multiple clients in a local deployment. This type of deployment is useful for fast development iteration.

Reviewer: I'm still uneasy about these sentences, but not your changes. Why the comma in the first sentence?

Reviewer: "Your game simulation with its own instance of the SpatialOS Runtime runs on your development machine. You can run multiple clients in a local deployment, which is useful for fast development iteration."

May be a better option?

Reviewer 2: Agree with Jo on this. Either we need to add a comma after "game simulation", or remove the comma that's currently there. I'd probably go for adding a comma, as otherwise the sentence is a little hard to parse.

I also like Jo's simplification of the second and third sentences into one.

So I'd recommend:

> Your game simulation, with its own instance of the SpatialOS Runtime, runs on your development machine. You can run multiple clients in a local deployment, which is useful for fast development iteration.

The **Spatial Networking** option is the networking switch; you use this to switch your game's deployment from native Unreal networking to SpatialOS networking.</br></br>

1. Now, run the game: in the **Modes** section of the window, select **New Editor Window (PIE)**.</br>

> You are now running one game server and two game clients in a local deployment in your Unreal Editor. You do not need to worry about the loading bar being stuck on 45%, this is just the shaders being compiled for the first time. (If it doesn't run, see **Note** below.)</br>

Reviewer: Worth saying how long (approx) they will be stuck?

Reviewer 2: Agree, this would be good if possible!

Author: Unfortunately, I don't think there is anyway of estimating the time, it will depend on the machine.

> You are now running one game server and two game clients in a local deployment in your Unreal Editor. You do not need to worry about the loading bar being stuck on 45%, this is just the shaders being compiled for the first time. (If it doesn't run, see **Note** below.)</br>

Reviewer 2: ```suggestion
You are now running one game server and two game clients in a local deployment in your Unreal Editor. You don't need to worry about the loading bar being stuck on 45%; this is just the shaders being compiled for the first time. (If it doesn't run, see **Note** below.)</br>
```

- **do not** -> **don't** (we tend to use contractions as they're more friendly/informal)
- comma -> semicolon (I find it hard to explain exactly why, but a semicolon (or dash) is more correct here.)
